### PR TITLE
Feature/landing page

### PR DESCRIPTION
**Pull request by 10dimensions**
- Setup a landing page with quick info
- Wallet Prompt Instructions
- Inline styling applied

To-Do: Wallet Network not determined after connection (Kovan testnet)

### CODE REVIEW

**Comment by codynhat** [comment]
I went ahead and merged the changes I just made to `develop`.

Only issue I see is the map is not fixed to the sides once the landing page is hidden. There is whitespace on the left and it is able to scroll horizontally. Does that happen for you?

**Comment by 10dimensions** [comment]
I saw an initial whitespace for the static content...
So pushed the left margin a bit.

I'm guessing, if it has to do with removal of the "Testnet" button on top left.

File: components/Map.js [hunk 1]
```diff
@@ -354,7 +354,7 @@ function Map({ adminAddress, adminContract, account, ceramic, ipfs }) {
           ceramic={ceramic}
           ipfs={ipfs}
         ></Sidebar>
-      ) : null}
+      ) : null}gridLayer
```
Comment (codynhat) [inline components/Map.js:357]: I think this is the cause. Is this here for any particular reason or just left over from debugging?

Comment (codynhat) [reply components/Map.js:357]: Removing `gridLayer` locally fixed the scrolling/whitespace issue for me.

Comment (10dimensions) [reply components/Map.js:357]: I think some IDE fault.. while debugging.

**Pull request merged**